frontend/src/pages/projects/projectSharing/ProjectSharingPermissionSelection.tsx

```diff
@@ -27,6 +27,7 @@ const ProjectSharingPermissionSelection: React.FC<ProjectSharingPermissionSelect
 
   return (
     <Select
+      data-testid={selection}
```

Comment:
Test IDs should have a static portion.
In fact here I don't think it should even be necessary to have a dynamic portion unless this selection is rendered multiple time adjacent to each other within the same scope.